packages/mds-agency/api.ts

```diff
@@ -484,7 +482,7 @@ function api(app: express.Express): express.Express {
       : rows
 
     const deviceIdSubset = deviceIdSuperset.slice(skip, skip + take).map(record => record.device_id)
-    const devices = (await cache.readDevices(deviceIdSubset)).reduce((acc: Device[], device: CacheReadDeviceResult) => {
+    const devices = (await db.readDeviceList(deviceIdSubset)).reduce((acc: Device[], device: Device) => {
```

Comment:
I think the issue here is going to be that the cache contains at least one additional property, `status`, that is not stored in the db.
```
hgetall device:13c6cb86-978c-4633-a830-411eab28e1f2:device
 1) "provider_id"
 2) "5f7114d1-4091-46ee-b492-e55875f7de00"
 3) "vehicle_id"
 4) "13c6cb86-978c-4633-a830-411eab28e1f2"
 5) "type"
 6) "scooter"
 7) "propulsion.0"
 8) "electric"
 9) "propulsion.1"
10) "human"
11) "year"
12) "2019"
13) "mfgr"
14) "ScootFast"
15) "model"
16) "T100"
17) "recorded"
18) "1566230943903"
19) "status"
20) "removed"
```

Reply:
I think this might be a non-issue. `getVehicles` also fetches events from the cache, and if it can't find a corresponding event for a given `device_id`, then it sets `status = removed`.

Reply:
Awesome.